### PR TITLE
Check for OS before invoking function

### PR DESCRIPTION
**Pull request by zeantsoi**
:eyeglasses: @bsimpson63

### CODE REVIEW

File: reddit_adzerk/adzerkpromote.py [hunk 1]
```diff
@@ -252,20 +252,18 @@ def update_flight(link, campaign, az_campaign):
     if campaign.mobile_os:
         queries_list = []
 
-        ios_targets = get_mobile_targeting_query(os_str='iOS',
-                                                 lookup_str='modelName',
-                                                 mobile_os=campaign.mobile_os,
-                                                 devices=campaign.ios_devices,
-                                                 versions=campaign.ios_version_range)
-        if ios_targets:
+        if 'iOS' in mobile_os:
```
Comment (bsimpson63) [inline reddit_adzerk/adzerkpromote.py]: does this need to be `campaign.mobile_os`?

**Comment by zeantsoi** [comment]
:eyeglasses: @dwick also.

Checking whether to run `get_mobile_targeting_query` before actually invoking it.

File: reddit_adzerk/adzerkpromote.py (hunk 1 above)
Comment (zeantsoi) [inline reddit_adzerk/adzerkpromote.py]: It does, committed.

File: reddit_adzerk/adzerk_utils.py [hunk 2]
```diff
@@ -65,26 +65,22 @@ def get_version_query(version_range):
 
 def get_mobile_targeting_query(os_str='',
                                lookup_str='',
-                               mobile_os=[],
                                devices=None,
                                versions=None):
     from adzerk_utils import get_version_query
```
Comment (bsimpson63) [inline reddit_adzerk/adzerk_utils.py]: what's up with this import?

**Comment by bsimpson63** [comment]
:fish:

**Comment by zeantsoi** [comment]
8beecda0d0bf0feb40116ebeeb7b5cc71b44937c

**Pull request closed without merging**